controllers/horizontal_runner_autoscaler_batch_scale.go

```diff
@@ -79,6 +79,7 @@ func (s *batchScaler) Add(st *ScaleTarget) {
 				for {
 					select {
 					case <-after:
+						// TODO: What was the intention here? Close the channel?
```

Comment:
Can you help me with this one?

```
controllers/horizontal_runner_autoscaler_batch_scale.go:83:7: ineffectual assignment to after (ineffassign)
                                                after = nil
```

Reply:
Ah good catch. This was emitted probably because `after` will never be read after it has been updated to `nil` due to that we call `break batch` immediately after the assignment.
I believe you can just remove it.